extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -459,9 +464,11 @@ export class SchemaCompareResult {
 		}).component();
 
 		this.applyButton.onDidClick(async (click) => {
+			Telemetry.sendTelemetryEvent('SchemaCompareApply');
 			let service = await SchemaCompareResult.getService('MSSQL');
 			let result = await service.schemaComparePublishChanges(this.comparisonResult.operationId, this.targetEndpointInfo.serverName, this.targetEndpointInfo.databaseName, azdata.TaskExecutionMode.execute);
 			if (!result || !result.success) {
+				Telemetry.sendTelemetryEvent('SchemaCompareApplyFailed');
```

Comment:
For all of these operations do you think it'd be useful to time them? Maybe instead of just a Failed event you could send a SchemaCompareApplyFinished and then include succeeded, durationMs and message properties.

Reply:
added start and finish time